lib/mumuki/domain/status/submission/submission.rb

```diff
@@ -23,8 +23,6 @@ def group
     self
   end
 
-  # Tells if a new, different submission should be tried.
```

Comment:
Why? :stuck_out_tongue:

Reply:
I think it added nothing. Plus, the line below this one is rendered useless with tests that'll always keep it updated.

But it can come back ~unlike others~ if you think so or we can toss a virtual coin

Reply:
:+1: